ui/dashboardApp/layout/main/Sider/index.module.less

```diff
@@ -19,6 +19,10 @@
       display: flex;
       flex-direction: column;
       padding-bottom: 20px;
+
+      .ant-menu.ant-menu-inline-collapsed {
+        width: @menu-collapsed-width; // 80px
```

Comment:
Fix the broken sider menu when it is collapsed.

![image](https://user-images.githubusercontent.com/1284531/144451095-bbddaa37-c255-46a8-aa05-5ef67a7ff21a.png)